include/cudaq/Frontend/nvqpp/ASTBridge.h

```diff
@@ -1,5 +1,5 @@
 /****************************************************************-*- C++ -*-****
- * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
```

Comment:
I assume someone will have a PR to update all the copyrights en masse, @bettinaheim ?

Reply:
Perhaps this merits further discussion, but I would expect that we would not update the copyright for a given year unless we actually change the file in the new year. That's how I've managed these types of things in the past anyway.

Reply:
Let's just update files as we go - using the dev container in VS Code is doing automatically.